src/Shared/Recruit.Shared.Web/TagHelpers/ConditionTagHelper.cs

```diff
@@ -15,4 +15,32 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
     }
+
+    [HtmlTargetElement(Attributes = "asp-hide")]
+    public class HideTagHelper : TagHelper
+    {
+        public bool AspHide { get; set; }
+
+        public override void Process(TagHelperContext context, TagHelperOutput output)
+        {
+            if (AspHide)
+            {
+                output.SuppressOutput();
+            }
+        }
+    }
+
+    [HtmlTargetElement(Attributes = "asp-show")]
+    public class ShowTagHelper : TagHelper
```

Comment:
I would think all the `asp-condition` should be replaced with this now and you would drop that tag helper.

Reply:
I spoke to @leewadhams  about making this change, we both agreed its going to have a huge impact so left it for later, as mentioned in my comment a tech debt task is in place and @mgwilliam is aware of it.